src/store/reducer/board.ts

```diff
@@ -13,8 +13,8 @@ export const boardReducer = (state: BoardState = {status: "unknown"}, action: Re
       };
     }
     case ActionType.EditBoard: {
-      // Moderator started voting phase - notification to moderator (user who started the voting)
-      if (state.data?.voting !== action.board.voting) {
+      // Moderator started voting phase - notification to moderator (user w″ho started the voting)
+      if (state.data?.voting !== action.board.voting && action.board.voting) {
```

Comment:
In theory `if (action.board.voting) {...}` should be enough. `action.board.voting` is undefined if it's not getting changed

Reply:
Yes, that works